manual/README.md

```diff
@@ -142,6 +142,16 @@ In particular, toplevel examples must end with a double semi-colon `;;`,
 otherwise an error would be raised.
 The `verbatim` does not require a final `;;` and is intended to be
 a lighter mode for code examples.
+If you want to declare a signature instead of ocaml code,
+you must use the `{signature}` argument to the `caml_example` environment.
+```latex
+\begin{caml_example*}{signature}
+module A : sig
```

Comment:
I would start the example with a value

```OCaml
val none : 'a option
```
since the keyword `val` is only used in signature and thus make it clear that this is signature example on the first word of the example, whereas using a module requires to read three tokens.